client/components/forms/form-radio/style.scss

```
@@ -1,3 +1,7 @@
+@import "@automattic/typography/styles/variables";
+// stylelint-disable-next-line scss/at-import-no-partial-leading-underscore
+@import "../../../assets/stylesheets/shared/_extends-forms.scss";
```

Could you try if this works? Looks cleaner:
```suggestion
@import "calypso/assets/stylesheets/shared/extends-forms";
```